src/controllers/server.js

```diff
@@ -215,6 +219,16 @@ class Server extends EventEmitter {
             },
         ], (err, reboot) => {
             if (err) {
+                if (err.statusCode === 404) { // container not found
+                    this.log.error('The container for this server could not be found. Trying to rebuild it.');
+                    this.emit('console', `${Ansi.style.red}(Daemon) The container of this server could not be found and has to be rebuilt.`);
```

Comment:
We can probably skip this console report, the rebuild will output information to the screen anyways, and this will probably scare users when in reality it will most likely not be a huge impact.